content/common/avalanche-ictt/setup-manual.mdx

```diff
@@ -94,15 +94,15 @@ Currency Symbol:   NAT
 
 From this output, take note of the following parameters
 
-- Funded Address (with 100 tokens),
+- Main funded account EWOQ (with 100 tokens),
 - Interchain Messaging Registry on C-chain, and
 - Interchain Messaging Registry on Avalanche L1
 
-Set these parameters as environment variables so that we can manage them easily and also use them in the commands later.
+**IMPORTANT!** Set these parameters as environment variables so that we can manage them easily and use them in commands later.
 
 ```bash
 export FUNDED_ADDRESS=<Funded Address (with 100 tokens)>
 export TELEPORTER_REGISTRY_C_CHAIN=<Interchain Messaging Registry on C-chain>
-export TELEPORTER_REGISTRY_SUBNET=<Interchain Messaging Registry on Avalanche L1>
+export TELEPORTER_REGISTRY_L1=<Interchain Messaging Registry on Avalanche L1>
```

Comment:
Not sure about this one, please try if it's always the same. If it is, lets also add to `devcontainer.json`